python/core/fundamental-data-structures/py-tuple-assignment.md

```diff
@@ -0,0 +1,114 @@
+---
+author: kapnobatai136
+
+tags:
+  - introduction
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+links:
+  - '[Tuples and Sequences](https://docs.python.org/3.5/tutorial/datastructures.html#tuples-and-sequences){website}'
+
+---
+
+# Tuple Assignment
+
+---
+## Content
+
+The **tuple assignment** feature allows packing and unpacking such that a tuple left of the assignment will be assigned all values from the tuple right of the assignment:
```

Comment:
hmm

`(fname, lname, age) = flash` 

>  The **tuple assignment** feature allows packing and unpacking such that a tuple left of the assignment will be assigned all values from the tuple right of the assignment: 

is ` (fname, lname, age)` considered a tuple here? I don't think so, right ?it's just syntax used to destructure (unpack) the tuple. not very sure on this one

Reply:
it is. you can assign a tuple to another tuple

![image](https://user-images.githubusercontent.com/19802557/80080556-b12f5f00-8549-11ea-8c5d-ec2d56a2d761.png)

Reply:
but I can't think of a reason you would do that

Reply:
I'll re-word it to be more explicit